invisible_cities/io/dst_io.py

```diff
@@ -24,3 +26,43 @@ def load_dst(filename, group, node):
 def load_dsts(dst_list, group, node):
     dsts = [load_dst(filename, group, node) for filename in dst_list]
     return pd.concat(dsts)
+
+class TableMismatchError(Exception):
+    def __init__(self):
+        s  = 'The table and dataframe dont match! '
+        Exception.__init__(self, s)
+
+def _store_pandas_as_tables(h5out, df, group_name, table_name, compression='ZLIB4', descriptive_string=None, str_col_length=32):
+    if len(df) == 0:
+        warnings.warn(f' dataframe is empty', UserWarning)
+    if '/'+group_name not in h5out:
+        group = h5out.create_group(h5out.root,group_name)
+    else:
+        group = getattr(h5out.root,group_name)
+    if table_name in group:
+        table=getattr(group,table_name)
+    else:
+        tabledef={}
+        for indx, colname in enumerate(df.columns):
+            coltype = df[colname].dtype.name
+            try:
+                tabledef[colname] = tb.Col.from_type(coltype, pos = indx)
+            except ValueError:
+                tabledef[colname] = tb.StringCol(32, pos = indx)
+
+        if descriptive_string is None:
+            descriptive_string = ''
```

Comment:
the argument `descriptive_string` could be `""` directly instead of `None` and `if None set it to ""`

Reply:
Probably in this case it doesnt matter but I prefer not to have mutable keyword arguments

Reply:
strings are not mutable :)

Reply:
oooops, true, my fault